interface/.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin
+build
+node_modules
```

Comment:
Which is to say yes, I think you may be right. If we are at the stage where Docker is handling the whole interface build process then `node_modules` could get cleaned out. Also: HOORAY for Docker builds!!! The primary reason our builds are dodgy is because of `npm`, so if we've got a Docker image containerizing that headache now I am very very very happy.

😄 😸 😃 😺 😄 😸 😃 😺 😄 😸 😃 😺 😄 😸 😃 😺 😄 😸 😃 😺 😄 😸 😃 😺